src/helpers/url.js

```diff
@@ -6,9 +6,9 @@ module.exports = {
     if (!event.headers) return null;
 
     return url.format({
-      protocol: event.headers['X-Forwarded-Proto'],
-      host: event.headers.Host,
-      pathname: event.path
+      protocol: event.headers['x-forwarded-proto'],
+      host: event.headers.host,
```

Comment:
Do we possibly know what could the value for these headers be?

Reply:
It comes from the AWS Lambda `event` object. You can't modify that with custom headers in the request as it will get overwritten internally between API GW and Lambda.